jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/security/DefaultUserIdentity.java

```diff
@@ -0,0 +1,72 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.ee10.servlet.security;
+
+import java.security.Principal;
+import javax.security.auth.Subject;
+
+import org.eclipse.jetty.security.DefaultIdentityService;
+import org.eclipse.jetty.security.UserIdentity;
+
+/**
+ * The default implementation of UserIdentity.
+ */
+public class DefaultUserIdentity implements UserIdentity
```

Comment:
what is this? Is it used anywhere?

Reply:
It is used for the test. The actual one is in an internal package, so I could make that one public if necessary. Wasn't sure if I should move it public just for a test.

Reply:
TestUserIdentity?

Reply:
Turns out I could remove its usage from the test, so now I have deleted this altogether.